source/code-generation/query-converter/convert-queries.txt

```diff
@@ -0,0 +1,158 @@
+.. _rm-convert-queries:
+
+===============
+Convert Queries
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can convert SQL queries to MongoDB syntax with the query converter. 
+Query converter accounts for your project's mapping rules and schema 
+transformations when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      From the :guilabel:`Code Generation` tab, click the 
+      :guilabel:`Query Converter` pane.
+
+   .. step:: Open the SQL query converter view
+
+      If it is your first time using the query converter in your 
+      project, click :guilabel:`Paste SQL Query`.
+      If you have previously converted SQL code in your project, click the
+      :guilabel:`+ ADD` button on the left-hand pane next to 
+      :guilabel:`Queries`.
+
+   .. step:: Enter your SQL query
+
+      a. Copy your SQL query to your clipboard.
+      #. Under the :guilabel:`SQL Query` header, paste your SQL query
+         into the text box.
+   
+   .. step:: Convert and test syntax
+
+      a. Click the :guilabel:`Convert` button. Wait for your SQL code to 
+         converted. The converted MongoDB syntax code displays 
+         in the :guilabel:`Converted MongoDB Query` text box.
+      #. Click the :icon-fa5:`copy` icon to copy the MongoDB syntax 
+         to your clipboard.
+      #. Test and validate the generated MongoDB syntax.
+
+      .. tip:: 
```

Comment:
We also allow people to rename queries (just so they can be identified in the query converter), and delete them (should you no longer want to see them in the project). These are relatively minor things so they could probably be added as optional steps in this topic, rather than have whole new topics.

Reply:
I added a tip in step 4.1 calling out this functionality.